src/main/java/com/bamdoliro/gati/domain/board/presentation/dto/request/CreateBoardRequestDto.java

```diff
@@ -18,16 +17,20 @@ public class CreateBoardRequestDto {
     @NotNull
     private Long communityId;
     @NotNull
+    @Size(min = 3, max = 20)
     private String title;
     @NotNull
+    @Size(min = 10, max = 4000)
     private String content;
 
     public Board toEntity(User writer, Community community) {
-        return Board.builder()
+        Board board = Board.builder()
                 .title(title)
                 .content(content)
                 .community(community)
                 .writer(writer)
                 .build();
+        board.setStatus(Status.EXISTED);
+        return board;
```

Comment:
```suggestion
        return Board.builder()
                .title(title)
                .content(content)
                .community(community)
                .writer(writer)
                .build();
```
create 할 때는 무조건 status가 EXISTED니까 굳이 setter안 쓰고 생성자에 넣어주면 될 듯 합니다. (-> 생성자에 parameter 추가 하지 않고 그냥 안에서 `this.status = Status.EXISTED`)